misc/python/materialize/mzcompose/composition.py

```diff
@@ -753,6 +754,17 @@ def sanity_restart_mz(self) -> None:
                 print("Service materialized not running, will not restart it.")
                 return
 
+            for (db,) in self.sql_query("SELECT name FROM mz_databases"):
```

Comment:
Maybe it will be more useful if it can be generalized as follows:
- use SHOW CREATE to capture the output for all eligible objects in a database
- drop the previous database create a new database with the same name but a `sanity_restart_` prefix and then recreate all objects that previously existed in ID order ,that is, in the other they were previously created (so that all dependencies are satisifed).

What this provides is mostly catalog verification, that is, that the stuff in `SHOW CREATE` can then be runnable against MZ , in addition for checking for any panics at creation time.

This however would be in scope outside of the scope of this PR.